contact.html

```diff
@@ -52,17 +52,22 @@
         </h1>
       <div class="field">
         <label>Name</label>
-        <input type="text" name="name" placeholder="Name">
+        <input type="text" name="name" placeholder="Name" required />
```

Comment:
Do we need to require their name? We can keep the form simpler without it or make it optional.